### DISCUSSION TITLE
Updating the property of a database file.

### DISCUSSION THREAD

**Discussion by dvsrk** [dataplat/dbatools#7298] [General]
I would like to update properties of the database file, something like file growth, max file size and name. Is there a way I could do that using any of the dbatools command?
I was looking in the below direction but not sure if it's the right approach, 

`
Get-DbaDatabase -SqlInstance . | 
where name -EQ 'Test01' | 
select -ExpandProperty FileGroups | 
select -ExpandProperty files | 
get-member | where name -EQ 'SetPropertyValue'
`

**Comment by potatoqualitee** [comment]
Hey @dvsrk - this command will do it for you. Generally, I use SMO (the DLLs that power SSMS) but in this case, it seems SMO was giving me issues so I used T-SQL

https://github.com/sqlcollaborative/dbadisa/blob/master/public/Set-DbsDbFileSize.ps1
